shell/browser/ui/message_box_gtk.cc

```diff
@@ -38,10 +40,14 @@ MessageBoxSettings::~MessageBoxSettings() = default;
 
 namespace {
 
+// <ID, messageBox> map
+std::map<std::string, GtkWidget*> g_dialogs;
```

Comment:
you can use `std::unordered_map`

Reply:
Chromium suggests avoiding `std::unordered_map` unless there is a very good reason:
https://chromium.googlesource.com/chromium/src/base/+/master/containers/#map-and-set-selection